### ISSUE TITLE
Bug: 10-7959f-2 submit in staging did not produce a success callback, but did produce a confirmation email

### ISSUE THREAD

**Issue by baltes** [department-of-veterans-affairs/va.gov-team#95556]

**Comment by michaelclement** [comment]
From what I can tell, this happened because technically `Not processed` is a valid Pega status (as opposed to a value of `nil`, which submissions have before we've actually made contact with Pega).

This leads to the following question: when will we see a status of not processed? Is that only while the Pega endpoint is under construction and so the forms have nowhere to go (as in the case of F2)? Or is there a non-zero chance that say, FMP1, could have this issue?

@baltes @andrea-merrill

**Comment by pshahVA** [comment]
Hey there. Is this happening to the F1 form that's already in prod?

**Comment by baltes** [comment]
@stevelong00 Can you summarize the intended fix here? Hoping to confirm again that the fix should be to handle status as it's reported appropriately in addition to checking for a non-nil status in the payload.

**Comment by stevelong00** [comment]
@baltes The intended fix is to add logic to check the Pega status and send a failure email instead of a confirmation email when the status is anything other than "processed".

**Comment by stevelong00** [comment]
[The PR](https://github.com/department-of-veterans-affairs/vets-api/pull/19599)

**Comment by baltes** [comment]
> [@baltes](https://github.com/baltes) The intended fix is to add logic to check the Pega status and send a failure email instead of a confirmation email when the status is anything other than "processed".

Just so we have a record of it, the goal is to avoid sending a false positive when processing was not successful. We do not need to send a failure when PEGA gives a response other than "processed" since the file could still be corrected.

**Comment by stevelong00** [comment]
@baltes The PR has just been merged.  This change is behind a feature flag, which I will enable in staging to test later today.

**Comment by baltes** [comment]
Do you have suggestions on how I can verify this in staging?

**Comment by stevelong00** [comment]
@baltes It looks like Pega has updated their workflow for 10-7959f-2 and no longer reports 'Not Processed' for these form submissions.  I don't know of another way to test easily.

@michaelclement and I just tested this via the rails console by simulating Pega reports.  With the fix disabled we saw the confirmation email come through as expected with the bug, and with the fix enabled we saw no confirmation email.
